pkg/pillar/types/volumetypes.go

```diff
@@ -178,6 +168,13 @@ func (status VolumeStatus) PathName() string {
 		strings.ToLower(status.ContentFormat.String()))
 }
 
+// PVCName returns the volume name for kubernetes(longhorn)
+func (status VolumeStatus) GetPVCName() string {
```

Comment:
Someone will be reading this code in 5 years and wonder why.
So please add back your comment that these can not contain special characters like e.g., "#"